packages/text-indexing/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@jbrowse/text-indexing",
+  "version": "1.6.7",
+  "description": "JBrowse 2 text indexing for desktop",
+  "keywords": [
+    "jbrowse",
+    "jbrowse2",
+    "bionode",
+    "biojs",
+    "genomics"
+  ],
+  "license": "Apache-2.0",
+  "homepage": "https://jbrowse.org",
+  "bugs": "https://github.com/GMOD/jbrowse-components/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GMOD/jbrowse-components.git",
+    "directory": "packages/text-indexing"
+  },
+  "author": "JBrowse Team",
+  "main": "dist/index.js",
+  "scripts": {
+    "test": "cd ../..; jest packages/text-indexing",
+    "coverage": "yarn test --coverage",
+    "lint": "tsc --noEmit && eslint --ext .js,.ts,.jsx,.tsx .",
+    "clean": "rimraf dist",
+    "prebuild": "yarn clean",
+    "prepack": "yarn build",
+    "build": "NODE_ENV=production babel src/ --out-dir dist/ --extensions '.ts,.js,.tsx,.jsx'"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.16.3",
+    "@material-ui/icons": "^4.0.1",
+    "abortable-promise-cache": "^1.1.3",
+    "base64-js": "^1.3.0",
```

Comment:
probably the text-indexing package itself doesn't need all these dependencies, looks maybe based on jbrowse/core

Reply:
removed unused dependencies